types/twilio-video/index.d.ts

```diff
@@ -569,8 +574,9 @@ export interface LocalDataTrackOptions {
 }
 export type LocalTrack = LocalAudioTrack | LocalVideoTrack | LocalDataTrack;
 export interface LocalTrackOptions {
-    logLevel: LogLevel | LogLevels;
+    logLevel?: LogLevel | LogLevels;
```

Comment:
```suggestion
    logLevel: LogLevel | LogLevels;
```

It's not optional according to docs?
https://media.twiliocdn.com/sdk/js/video/releases/2.3.0/docs/global.html#LocalTrackOptions